VideoLocker/src/main/java/org/edx/mobile/module/notification/EdxLocalParseChannel.java

```diff
@@ -0,0 +1,45 @@
+package org.edx.mobile.module.notification;
+
+import java.io.Serializable;
+
+/**
+ * need a place to hold the logic of mapping courseId to channelId.
+ */
+public class EdxLocalParseChannel implements Serializable{
+
+    private String courseId;
+    private String channelId;
+    private boolean subscribed;
+
+    public EdxLocalParseChannel(){}
+
+    public EdxLocalParseChannel(String courseId, boolean subscribed){
+        this.courseId = courseId;
+        this.channelId = courseId.replaceAll("[^A-Za-z0-9]", "");
```

Comment:
Is this just temporary? We need to agree on what the channel mapping for a course id is.